fem/transfer.cpp

```diff
@@ -462,6 +473,7 @@ void L2ProjectionGridTransfer::L2ProjectionL2Space::MultTranspose(
 void L2ProjectionGridTransfer::L2ProjectionL2Space::Prolongate(
    const Vector &x, Vector &y) const
 {
+   MFEM_VERIFY(P.Size() > 0, "Prolongation not supported for these spaces.")
```

Comment:
We need to handle the case when the local mesh is empty. Right now this check will fail, I think.

Reply:
Does this look OK to you?

```diff
diff --git a/fem/transfer.cpp b/fem/transfer.cpp
index aaf0954ae..40b34312f 100644
--- a/fem/transfer.cpp
+++ b/fem/transfer.cpp
@@ -473,6 +473,7 @@ void L2ProjectionGridTransfer::L2ProjectionL2Space::MultTranspose(
 void L2ProjectionGridTransfer::L2ProjectionL2Space::Prolongate(
    const Vector &x, Vector &y) const
 {
+   if (fes_ho.GetNE() == 0) { return; }
    MFEM_VERIFY(P.Size() > 0, "Prolongation not supported for these spaces.")
    int vdim = fes_ho.GetVDim();
    Array<int> vdofs;
@@ -509,6 +510,7 @@ void L2ProjectionGridTransfer::L2ProjectionL2Space::Prolongate(
 void L2ProjectionGridTransfer::L2ProjectionL2Space::ProlongateTranspose(
    const Vector &x, Vector &y) const
 {
+   if (fes_ho.GetNE() == 0) { return; }
    MFEM_VERIFY(P.Size() > 0, "Prolongation not supported for these spaces.")
    int vdim = fes_ho.GetVDim();
    Array<int> vdofs;
```

Reply:
Yes, this should do it.

Reply:
Done